### DISCUSSION TITLE
Changing background color

### DISCUSSION THREAD

**Discussion by persius77** [lllyasviel/Fooocus#2718] [Q&A] [answered]
I've noticed that some Fooocus users have a nice dark gray background.  Mine opens as white.   How can I change my settings so that Fooocus opens with a gray background? 

Thank you.

**Comment by ILikeToasters** [comment, marked as answer]
You are probably talking about dark mode. Myself I run my browser in dark mode for everything and it is a setting in browsers or you can add  `--theme dark` to your .bat file from what I have seen.

**Reply by aolko** [reply]
It adapts to your browser preference, which adapts to your OS preference

**Reply by ILikeToasters** [reply]
Yes. But also can be done without changing the browser or OS.
